bundles/GraphicServiceOrder/Service/OrderService.php

```diff
@@ -213,6 +219,10 @@ private function updateUserWithGSOrder(GsOrder $gsOrder)
      */
     public function createOrder(GsOrder $gsOrder, $form)
     {
+        $gsOrder->setIssueId(0);
+        $gsOrder->setIssueKey(0);
```

Comment:
I suppose you set these because they cannot be null. If so, then consider adding a comment about this.

Reply:
Added comment